include/tvm/relay/error.h

```diff
@@ -20,13 +20,11 @@ struct InternalError : Error {
   explicit InternalError(const std::string &msg) : Error(msg) {}
 };
 
-// TODO(@jroesch): we should change spanned errors to report
-// errors against the Environment, inverting control to error definition.
-struct FatalTypeError : dmlc::Error {
-  explicit FatalTypeError(const std::string &s) : dmlc::Error(s) {}
+struct FatalTypeError : Error {
+  explicit FatalTypeError(const std::string &s) : Error(s) {}
 };
 
-struct TypecheckerError : public dmlc::Error {
+struct TypecheckerError : Error {
```

Comment:
should it be explicit public inheritance? private inheritance will cause it not to be recognized as dmlc::error

Reply:
I think it is good to always be explicit